mk_verificator/tests/system/mirgation/test_mirgation.py

```diff
@@ -0,0 +1,66 @@
+import novaclient.client as client
+import mk_verificator.utils as utils
+
+import time
+
+
+def wait_for_vm_status_is_active(client, vm_id, retry_delay=5, timeout=60):
+    timeout_reached = False
+    start_time = time.time()
+    while not timeout_reached:
+        vm_status = client.servers.find(id=vm_id).status
+
+        if vm_status == 'ACTIVE':
+            break
+        elif vm_status == 'ERROR':
+            assert False, "VM is in the status 'error'"
+
+        time.sleep(retry_delay)
+        timeout_reached = (time.time() - start_time) > timeout
+    else:
+        assert vm_status == 'ACTIVE', \
+               "VM didn\'t get to status active, " \
+               "VM is in the status {}".format(vm_status)
+
+
+def checking_vm_host(client, vm_id, target_host):
+    wait_for_vm_status_is_active(client, vm_id)
+    host = client.servers.find(id=vm_id)._info['OS-EXT-SRV-ATTR:host']
+
+    assert host == target_host, "VM didn't migrate"
+
+
+def test_migration():
+    config = utils.get_configuration(__file__)
+
+    user_name = config['admin_username']
+    user_password = config['admin_password']
+    project_id = config['admin_project_id']
+    url = config['url']
+    endpoint_type = config['endpoint_type']
+    image_id = config['image_ref']
+    flavor_id = config['flavor_ref']
+    zone = config['availability_zone']
+    net_id = config['network_ref']
+
+    nova_client = client.Client(
+        '2.1', user_name, user_password, project_id, url,
+        service_type="compute", endpoint_type=endpoint_type)
+
+    nics = [{"net-id": net_id, "v4-fixed-ip": ''}]
+    vm = nova_client.servers.create(
+        'test_migrtion', image_id, flavor_id, nics=nics)
+
+    list_host = [host.host for host in nova_client.hosts.list(zone)]
+
+    list_host.remove(
+        nova_client.servers.find(id=vm.id)._info['OS-EXT-SRV-ATTR:host'])
+
+    wait_for_vm_status_is_active(nova_client, vm.id)
+
+    for host in list_host:
+        vm.live_migrate(host=host)
+        checking_vm_host(nova_client, vm.id, host)
+        print 'next host is {}'.format(host)
```

Comment:
please, don't use print. I would suggest to use array, let us say, 

broken_computes = [] 
for host in hosts:
   ...
   if failed:
      broken_computes.append(host)

assert not broken_computes, "Blah blah..."